v2/emailpassword/common-customizations/usermetadata/overrides.mdx

```diff
@@ -0,0 +1,80 @@
+---
+id: overrides
+title: Store data
```

Comment:
The title here should be `Overriding functions` (and same for all other recipes)